webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -51,6 +51,24 @@ export const clearSpaces = string => string.trim().replace(/\s\s+/, ' ');
 export const getDisplayName = Component =>
   Component.displayName || Component.name || 'Component';
 
+export const urlBuilder = (controller, action, id = undefined) =>
+  `/${controller}/${id ? `${id}/` : ''}${action}`;
```

Comment:
why is this required? this is missing a test. maybe we should create a url helper file instead of putting it all under helpers.

Reply:
added a  urlHelpers file